server/pkg/builtin/manifest_ja.yml

```diff
@@ -136,7 +136,7 @@ extensions:
             description: 有効にすると、カメラは地形の中を移動でき、地下エリアに入ることができます。
           fov:
             title: 視野（FOV）
-            description: カメラの視野 (FOV) (ラジアン単位)。カメラに見えるシーンの垂直方向の角度範囲を定義します。私たちのシステムでサポートされている範囲は 0.1 ～ 2.0 です。
+            description: "垂直方向のカメラの視野を変更します（単位はラジアン）。最小値: 0.1 最大値: 2.0"
```

Comment:
```suggestion
            description: 垂直方向のカメラの視野を変更します（単位はラジアン）。最小値: 0.1 最大値: 2.0
```

Reply:
@airslice if i remove  double quote("") it will show error message because of full colon(:)

Reply:
Ah got it, thank you!